public/app/features/alerting/unified/insights/mimir/perGroup/RulesPerGroupScene.tsx

```diff
@@ -18,8 +18,7 @@ export function getRulesPerGroupScene(timeRange: SceneTimeRange, datasource: Dat
   });
 
   return new SceneFlexItem({
```

Comment:
In case we want to use the same style in all scene flex items, maybe we could re-use the code somewhow to avoid having to repeat the updates in the style in several places? something like:
```

  const SCENE_FLEX_ITEM_STYLE = {
    minHeight: 300
  }

  return new SceneFlexItem({
    ...SCENE_FLEX_ITEM_STYLE,
    body: PanelBuilders.timeseries()
      .setTitle(panelTitle)
      .setData(query)
      .setCustomFieldConfig('drawStyle', GraphDrawStyle.Line)
      .build(),
  });
```

Reply:
Good idea, done!